### PR TITLE
Shake sensor

### PR DESCRIPTION
**Pull request by TheErry**
My reflections on how this week's project turned out:
This weeks project turned out quite good. I ran into some bugs and couldn't get Expo's debug menu to stop popping up whenever I shake the phone. I wanted to import other fonts too but it seem to be a bit tedious and I was very short on time this week. So I'm satisfied with how it is.

Things I'd like to have clarified or explained in more detail:
I wanted to add a home screen with a  logo and start button to make it more game like but had problems with the react native package for some reason, so I skipped it. I think the logo is still in the project maps however.

### CODE REVIEW

**Review by KarolCislowski — CHANGES_REQUESTED** [review]
You need to update the SDK used in your project to make it run on the actual version of the expo go.

**Comment by TheErry** [comment]
I updated Expo in the project if that helps?

File: README.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+
+If I had more time I would like to add a home screen with a logo, short rules and a start button. I tried to do that but had some problems with installing and running the packages needed, so I have to come back to it later. I also would like to have a high score list.
 
 ## View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+This is the project on Expo https://expo.io/@erry/projects/shake-it The Expo Go app is required to scan the QR code and open the project.
```
Comment (KarolCislowski) [inline README.md:13]: Did you connect your updated project to this link?? because I still have a prompt about the old version, when I try to open it?

**Review by KarolCislowski — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (TheErry) [reply README.md:13]: Yes, I re-deployed it to see if that helped. I don't get an error on this and didn't for the first deployment either?

**Comment by TheErry** [comment]
I actually got the error message now too. But this time I'm sure I've managed to update to the lates Expo SDK (v 41), and updated and installet the dependencies that needed it. I re-published the project and now it seems to work :)

File: README.md (hunk 1 above)
Comment (TheErry) [reply README.md:13]: This time I'm sure it's correctly updated :)

**Review by KarolCislowski — APPROVED** [review]
Great APP 🎉

**Pull request closed without merging**